### DISCUSSION TITLE
Removing "Dynamic Icons" after v0.145

### DISCUSSION THREAD

**Discussion by bourbonbourbon** [seerge/g-helper#2003] [General]
@seerge I see that you have removed the IconHelper code from Settings.cs in this commit ProArt backlight control support #1876 , Added CPU name to Fans+Power screen #1883 hash - d05298030fa29e8f65b61d500da7d2e8670d2e6d. Did you have some problems with it?

**Comment by seerge** [comment]
@bourbonbourbon hello.

I did, cause it was causing some UI issues on devices where windows taskbar hides or follows touch-screen rules (i.e. ROG Ally). It was still showing that single dot in a taskbar instead of a proper icon.

I have removed it for now, until I find a better solution :)
